pandas/core/dtypes/dtypes.py

```diff
@@ -1232,3 +1233,91 @@ def __from_arrow__(
             results.append(iarr)
 
         return IntervalArray._concat_same_type(results)
+
+
+@register_extension_dtype
+class DateDtype(PandasExtensionDtype):
+    """
+    An ExtensionDtype to hold a single date.
+    The attributes name & type are set when subclasses are created.
+    """
+
+    _date_aliases = {"date", "date64"}
+    _unit = "D"
+    _numpy_dtype = np.dtype("datetime64[D]")
+
+    def __str__(self):
+        return "date"
+
+    @property
+    def name(self) -> str_type:
+        """
+        The alias for DateDtype is ``'string'``.
+        """
+        return str(self)
+
+    @property
+    def type(self):
+        return Timestamp
```

Comment:
Is this an appropriate scalar type? Why not `datetime.date`?

Reply:
@TomAugspurger No it's not, I didn't use datetime.date because I thought there should be a cython implemented data structure rather than the native python one, so that's currently a place holder

Reply:
I don't think the scalar type is ever used in cython code.

Reply:
In _libs/tslib.pyx is the Timestamp implementation which is written in cython, I figured a similar thing might need to be done for date

Reply:
Ok yeah, fair enough, if it's already implemented in C, then I can change it